src/components/map-attribution/map-attribution.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useI18nStore } from "../../i18n/i18n-store.ts";
+
+const MapAttribution: React.FC = () => {
+  const i18n = useI18nStore().i18n();
+
+  const links = [
+    i18n.map.attribution.mapbox,
+    i18n.map.attribution.openStreetMap,
+    i18n.map.attribution.improve,
+    i18n.map.attribution.feedback,
+    i18n.map.attribution.imprint,
+    i18n.map.attribution.privacy,
+  ];
+
+  return (
+    <p
+      className={`
+      pointer-events-auto absolute bottom-[4.3rem] right-1 max-w-60 
```

Comment:
@Jaszkowic I did it with absolute here bc I wasn't sure how to make it with grid. Could you help me with that?